src/deferred/exceptionHook.js

```diff
@@ -11,10 +11,10 @@ var rerrorNames = /^(Eval|Internal|Range|Reference|Syntax|Type|URI)Error$/;
 
 jQuery.Deferred.exceptionHook = function( error, stack ) {
 
-	// Support: IE 8 - 9 only
```

Comment:
Let's not change this comment. We've decided some time ago to not remove info about workarounds for now non-supported browsers as long as we still need the workaround for other, supported ones.